en/docs/deploy-and-publish/publish-on-dev-portal/third-party-api-support.md

```diff
@@ -0,0 +1,38 @@
+# Third-party API support in WSO2 API Manager
+
+WSO2 API Manager provides the capability to publish APIs that are deployed in an external gateway. Since these APIs are not deployed in the WSO2 API gateway, the API will not be proxied through the gateway.
+
+In previous versions of API Manager, third-party APIs were available for only APIs deployed in the [AWS API Gateway]({{base_path}}/deploy-and-publish/publish-on-dev-portal/publish-aws-apis-in-the-dev-portal/) and when [publishing through external developer portals]({{base_path}}/deploy-and-publish/publish-on-dev-portal/publish-to-multiple-external-api-stores/).
```

Comment:
shall we change to " when Importing APIs from AWS API Gateway to WSO2 API Manager" and use the same phrasing in the docs